code/datums/ai/basic_mobs/targetting_datums/basic_targetting_datum.dm

```diff
@@ -15,6 +15,8 @@
 /datum/targetting_datum/basic
 	/// When we do our basic faction check, do we look for exact faction matches?
 	var/check_factions_exactly = FALSE
+	/// Only hostile against mobs when damaged?
+	var/check_hostile = FALSE
```

Comment:
I would use the `/datum/element/ai_retaliate` element and `/datum/ai_planning_subtree/target_retaliate` subtree for this, see the implementation on Pigs.

Reply:
I agree with Jacquerel.